### PR TITLE
fix!: Remove deprecated CSV plugin

### PR DESCRIPTION
**Pull request by hermanschaaf**
The CSV plugin was deprecated a couple of months ago, please use the `file` plugin instead.

Closes https://github.com/cloudquery/cloudquery/issues/10102

### CODE REVIEW

File: .release-please-manifest.json [hunk 1]
```diff
@@ -27,8 +27,6 @@
   "plugins/destination/postgresql+FILLER": "0.0.0",
   "plugins/destination/test": "1.3.26",
   "plugins/destination/test+FILLER": "0.0.0",
-  "plugins/destination/csv": "1.2.2",
```
Comment (candiduslynx) [inline .release-please-manifest.json:30]: should be left as is (used in the website to show the latest version)

**Review by candiduslynx — CHANGES_REQUESTED** [review]

File: .release-please-manifest.json (hunk 1 above)
Comment (hermanschaaf) [reply .release-please-manifest.json:30]: It's being removed from the website

**Review by candiduslynx — APPROVED** [review]

File: .release-please-manifest.json (hunk 1 above)
Comment (erezrokah) [reply .release-please-manifest.json:30]: Also it's not used in the Website I think. We use https://github.com/cloudquery/cloudquery/blob/845a8a0f344d253534eceeee4b9320658f9652d4/website/versions/destination-csv.json

**Review by erezrokah — APPROVED** [review]
🧹

**Pull request merged**